test/extensions/filters/network/dubbo_proxy/mocks.h

```diff
@@ -154,6 +154,14 @@ class MockRoute;
 
 namespace DubboFilters {
 
+class MockFilterChainFactory : public FilterChainFactory {
+public:
+  MockFilterChainFactory();
+  ~MockFilterChainFactory();
```

Comment:
mark this `override` to pass clang-tidy CI.

Reply:
done